roomserver/storage/sqlite3/redactions_table.go

```diff
@@ -118,6 +118,6 @@ func (s *redactionStatements) MarkRedactionValidated(
 	ctx context.Context, txn *sql.Tx, redactionEventID string, validated bool,
 ) error {
 	stmt := sqlutil.TxStmt(txn, s.markRedactionValidatedStmt)
-	_, err := stmt.ExecContext(ctx, redactionEventID, validated)
```

Comment:
Why was this swapped?

Reply:
The query also changed, since SQLite doesn't seem to like `$2` to come before `$1`.
The query now has `$1` for `validated` and `$2` for `redactionEventID`.

Before the change `markRedactionValidatedStmt` wouldn't update the table.